.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
 /mass-pr-workspace
```

Comment:
mass-pr-workspace works kinda as a tmp directory already, you could reuse that one